packages/components/src/MultiSelect/MultiSelect.css

```diff
@@ -0,0 +1,13 @@
+.multiSelectContainer {
+  position: relative;
+  width: 100%;
+}
+
+.multiSelectContainer:hover,
+.multiSelectContainer input {
+  cursor: pointer;
```

Comment:
```suggestion
  cursor: pointer;
  caret-color: transparent;
```

Reply:
Thank you for the suggestion, Rodrigo. It was implemented with the outside click